frontend/packages/patternfly/src/components/notification-drawer/notification-entry.tsx

```diff
@@ -97,4 +114,10 @@ type NotificationIconTypes = {
   type: NotificationTypes;
 };
 
+type NotificationActionProps = {
+  drawerToggle: (event: React.MouseEvent<HTMLElement>) => void;
```

Comment:
This should be `toggleDrawer`, but I think something like `onClick` would be better. The action shouldn't know or care what happens on click, just that there's a callback.

```suggestion
  onClick: (event: React.MouseEvent<HTMLElement>) => void;
```